CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+
+# Contributing
+
+Contributions to this plugin are welcome. To contribute a feature or change to this plugin, please follow the steps below
+
+1. Open an issue to describe your proposed improvement or feature, or describe an issue with the plugin
+2. Fork this respository
+3. Create your feature or fix branch (`git checkout -b my-new-feature`)
+4. If applicable, add a [CHANGELOG.md entry](#changelog) describing your change.
```

Comment:
```suggestion
4. Add a [CHANGELOG.md entry](#changelog) to describe your change.
```

Reply:
If we say "If applicable" I think we need to tell them how to figure out if it's applicable.